tests/js/instrumentedEnv/userEventIntegration.ts

```diff
@@ -0,0 +1,51 @@
+import {Hub} from '@sentry/hub';
+import {fill, isThenable, loadModule} from '@sentry/utils';
+
+const moduleName = '@testing-library/user-event';
+
+export function instrumentUserEvent(getCurrentHub: () => Hub): void {
+  const pkg = loadModule(moduleName) as any;
```

Comment:
Is the typing wrong on this? Why do you have to any it?

Reply:
The return type here is a generic and varies depending on the package itself. I opted to just type it as `any` for simplicity.

Reply:
is it actually wrong though?

Reply:
if i remove it, i get a `Object is of type 'unknown'` on the next line at `pkg.default`

Reply:
👍 

Talked on slack.

https://github.com/getsentry/sentry-javascript/blob/f042723bb442193f03d61d4d316cc2645aad6cea/packages/utils/src/node.ts#L46-L63

This isn't typed with respect to the name of the module so it doesn't understand how to resolve the types. 🤷